bin/nib-ls

```diff
@@ -214,18 +219,24 @@ def proc_file(f, opts):
         else:
             row += ['error']
 
-    if opts.stats:
+    if opts.stats or opts.counts:
         # We are doomed to load data
         try:
             d = vol.get_data()
             if not opts.stats_zeros:
                 d = d[np.nonzero(d)]
-            # just # of elements
-            row += ["[%d] " % np.prod(d.shape)]
-            # stats
-            row += [len(d) and '%.2g:%.2g' % (np.min(d), np.max(d)) or '-']
+            if opts.stats:
+                # just # of elements
+                row += ["@l[%d]" % np.prod(d.shape)]
+                # stats
+                row += [len(d) and '@l[%.2g, %.2g]' % (np.min(d), np.max(d)) or '-']
+            if opts.counts:
+                counter = Counter()
```

Comment:
scipy seems to be an optional dependency.... if decision is made not to drop python 2.6 then I will look into it, but otherwise better keep it leaner ... I could look into redoing it with pure numpy or dict I guess if needed.